src/cascadia/LocalTests_TerminalApp/TestHostApp/TestHostApp.vcxproj

```diff
@@ -0,0 +1,148 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Label="Globals">
+    <ProjectGuid>{A021EDFF-45C8-4DC2-BEF7-36E1B3B8CFE8}</ProjectGuid>
+    <RootNamespace>TestHostApp</RootNamespace>
+    <DefaultLanguage>en-US</DefaultLanguage>
+    <MinimumVisualStudioVersion>14.0</MinimumVisualStudioVersion>
+    <AppContainerApplication>true</AppContainerApplication>
+    <ApplicationType>Windows Store</ApplicationType>
+    <WindowsTargetPlatformMinVersion>10.0.18362.0</WindowsTargetPlatformMinVersion>
+    <WindowsTargetPlatformVersion>10.0.18362.0</WindowsTargetPlatformVersion>
+    <ApplicationTypeRevision>10.0</ApplicationTypeRevision>
+    <OutputSubDir>Tests\Data</OutputSubDir>
+    <UseWmXml>true</UseWmXml>
+    <ConfigurationType>Application</ConfigurationType>
+    <NoOutputRedirection>true</NoOutputRedirection>
+
+    <!--
+    These two properties are very important!
+    Without them, msbuild will stomp MinVersion and MaxVersionTested in the
+    Package.appxmanifest and replace them with whatever our values for
+    TargetPlatformMinVersion and TargetPlatformVersion are.
+     -->
+    <AppxOSMinVersionReplaceManifestVersion>false</AppxOSMinVersionReplaceManifestVersion>
+    <AppxOSMaxVersionTestedReplaceManifestVersion>false</AppxOSMaxVersionTestedReplaceManifestVersion>
+
+  </PropertyGroup>
+
+  <Import Project="$(SolutionDir)\common.openconsole.props" Condition="'$(OpenConsoleDir)'==''" />
+
+  <Import Project="$(OpenConsoleDir)\src\common.build.pre.props" />
+
+  <!-- This project is _heavily_ cribbed directly from the TAEF samples. In
+  order to avoid breaking this project, it's been left largely unmodified. The
+  only modifications are those near the bottom of the file:
+  * References to our dependent winrt projects (Connection, Settings, Control,
+    App)
+  * Manual copy steps to copy the actual test code (TerminalApp.LocalTests.dll)
+    and the dlls that TerminalConnection is dependent upon, but don't roll up
+    here for whatever reason.
+  -->
+
+  <PropertyGroup Label="UserMacros">
+    <GenerateAppxPackageOnBuild>false</GenerateAppxPackageOnBuild>
+  </PropertyGroup>
+
+  <ItemDefinitionGroup>
+    <ClCompile>
+      <AdditionalOptions>/bigobj %(AdditionalOptions)</AdditionalOptions>
+      <DisableSpecificWarnings>4453;28204;%(DisableSpecificWarnings)</DisableSpecificWarnings>
+      <AdditionalIncludeDirectories>%(AdditionalIncludeDirectories)
+        ;$(IntermediateOutputPath)
+      </AdditionalIncludeDirectories>
+      <PreprocessorDefinitions>INLINE_TEST_METHOD_MARKUP;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+    </ClCompile>
+    <Link>
+      <AdditionalDependencies>$(AdditionalDependencies);windowsapp.lib</AdditionalDependencies>

```

Comment:
is it possible to use the post props instead of having to add lib dependencies and all that jazz again?

Reply:
okay I get it, you DID do that, ignore me

Reply:
We can probably drop this entire link block and many of the ClCompile options because post sets them :smile: